apps/website/src/data/murals.ts

```diff
@@ -22,17 +21,28 @@ interface LiveMural extends CoreMural {
 
 type Mural = StaticMural | LiveMural;
 
+const gridCache: Record<string, Promise<Grid>> = {};
+
 const murals = {
   one: {
     name: "Pixel Project",
+    description:
+      "Explore the institute mural featuring 10,000 pixels unlocked by generous donors, raising $1,000,000 to fund the initial development of the Alveus Research & Recovery Institute.",
     type: "static",
-    grid: muralV1Grid as Grid,
+    get grid() {
+      return (gridCache["one"] ??= import("@/data/pixel-project-grid.json"));
```

Comment:
Switched this to an async import so that it is a separate chunk that doesn't need to be loaded on the Pixel Project 2 page (and so I can import this data file into the Next.js config)